SwanProjects/src/ProjectWidget.tsx

```diff
@@ -0,0 +1,205 @@
+    const newRelease = Object.keys(options.stacks_options[newStack])[0];
+    setRelease(newRelease);
+    const newPlatform = options.stacks_options[newStack][newRelease][0];
+    setPlatform(newPlatform);
+  };
+
+  const onChangeRelease = (event: React.ChangeEvent<{ value: unknown }>) => {
+    const newRelease = event.target.value as string;
+    setRelease(newRelease);
+    const newPlatform = options.stacks_options[stack][newRelease][0];
+    setPlatform(newPlatform);
+  };
+
+  const onChangePlatform = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setPlatform(event.target.value as string);
+  };
+
+  const onChangeUserScript = (
+    event: React.ChangeEvent<HTMLTextAreaElement>
+  ) => {
+    setUserScript(event.target.value as string);
+  };
+
+  return (
+    <div className="sw-Dialog-content">
+      <div className="sw-Dialog-project-name">
+        <swanProjectIcon.react className="sw-Dialog-project-icon" tag="span" />
+        <TextField
+          label="Project Name"
+          variant="outlined"
+          onChange={onChangeProjectName}
+          className="sw-Dialog-project-name-input"
+          size="small"
+          required
+          value={projectName}
+        />
+      </div>
+      <div className="sw-Dialog-select-stack">
+        <Card
+          label="LCG"
+          icon={sftIcon}
+          updateCallback={() => onChangeStack('LCG')}
+          isSelected={stack === 'LCG'}
+        />
+        <Card
+          label="CMSSW"
+          icon={cmsIcon}
+          updateCallback={() => onChangeStack('CMSSW')}
+          isSelected={stack === 'CMSSW'}
+        />
+      </div>
+      <div className="sw-Dialog-stack-options">
+        <div>
+          <div className="sw-Dialog-stack-option-label">
+            <span> Release </span>
+            <span className="sw-Dialog-release-tooltip">
+              <HelpTooltip
+                id="release"
+                message="Software stack: TODO! <br/> \
+                                  No message yet.<br/> \
+                                  ;("
+              />
+            </span>
+          </div>
+          <TextField
+            select
+            variant="outlined"
+            value={release}
+            onChange={onChangeRelease}
+            size="small"
+            fullWidth
+          >
+            {availableReleases.map((release, index) => {
+              return (
+                <MenuItem value={release} key={index}>
+                  {release}
+                </MenuItem>
+              );
+            })}
+          </TextField>
+        </div>
+        <div>
+          <div className="sw-Dialog-stack-option-label">
+            <span> Platform </span>
+            <span className="sw-Dialog-release-tooltip">
+              <HelpTooltip
+                id="release"
+                message="Platform: TODO! <br/> \
+                                  No message yet.<br/> \
+                                  ;("
```

Comment:
Same as above, what's the purpose?

Reply:
same as above.

Reply:
"Properties related to architecture and compiler. For advanced users" ?

Reply:
I think this might be different depending on CMSSW - LCG - FCC? Can you say here Omar what you show in this box for each? Right now for LCG we say "Platform" in the web form.

Reply:
Remember we have the stack, that is the icon,(LCG, CMSSW etc..)
every stack has releases and every release supports different platforms.
Let's take a look at the image.

![projectdialog](https://user-images.githubusercontent.com/275148/133576974-a15b3282-ade9-44ae-b2c4-491b99b3f988.png)



I was thinking of a generic message like: "The release of the software stack to associate to the container"
in the future, we can put a link to the documentation we have about the stacks we support.

Reply:
> I was thinking on a generic message like: "The release of the software stack associate to the container"

But this is for the Platform, no?

Here I would just say "OS, architecture and compiler version".

Reply:
yes, it is by project :D 

I like it!